--- origsrc/SystemTrayN/AppRun	2007-07-21 06:37:23.000000000 -0500
+++ src/SystemTrayN/AppRun	2009-10-20 01:50:52.014804300 -0500
@@ -11,7 +11,7 @@ if [ "x$PLATFORM" = x ]; then
 	i?86) ARCH=ix86 ;;
   esac
 
-  PLATFORM=`uname -s`-$ARCH export PLATFORM
+  PLATFORM=CYGWIN-$ARCH ; export PLATFORM
 fi
 
 if [ "x$MAKE" = x ]; then
--- origsrc/SystemTrayN/AppletRun	2006-01-21 08:57:22.000000000 -0600
+++ src/SystemTrayN/AppletRun	2009-10-20 01:50:55.564809200 -0500
@@ -11,7 +11,7 @@ if [ "x$PLATFORM" = x ]; then
 	i?86) ARCH=ix86 ;;
   esac
 
-  PLATFORM=`uname -s`-$ARCH export PLATFORM
+  PLATFORM=CYGWIN-$ARCH ; export PLATFORM
 fi
 
 if [ "x$MAKE" = x ]; then
